app/src/main/java/org/openobservatory/ooniprobe/common/ResubmitTask.java

```diff
@@ -133,11 +133,6 @@ protected Boolean doInBackground(Integer... params) {
                     EngineProvider.get().getDefaultSessionConfig(
                             getActivity(), BuildConfig.SOFTWARE_NAME, BuildConfig.VERSION_NAME, logger, proxy)
             );
-            // Updating resources with no timeout because we don't know for sure how much
-            // it will take to download them and choosing a timeout may prevent the operation
-            // to ever complete. (Ideally the user should be able to interrupt the process
-            // and there should be no timeout here.)
-            session.maybeUpdateResources(session.newContext());
```

Comment:
The `maybeUpdateResources` API call has been a stub for quite some time. Removing it is harmless.